crypt/crypt.go

```diff
@@ -44,6 +44,8 @@ var defaultOptions = Defaults{}
 var Parser = flags.NewParser(&defaultOptions, flags.Default)
 
 var nonceLength = 12
```

Comment:
are these expected to change length at all?
I also can't do multi line suggestions so ill do them one by one
```suggestion
const nonceLength = 12
```

Reply:
fixed in a3388f9